pkg/inventory/fake-inventory-client.go

```diff
@@ -96,14 +107,20 @@ func (fic *FakeClient) ClearError() {
 	fic.Err = nil
 }
 
-func (fic *FakeClient) GetClusterInventoryInfo(Info) (*unstructured.Unstructured, error) {
-	return nil, nil
+type FakeInventory struct {
+	BaseInventory
```

Comment:
Can BaseInventory be used instead of FakeInventory?

Reply:
Nevermind. BaseInventory doesn't impl the Info method. So it can't be used as the fake.